dietpi/dietpi-ddns

```diff
@@ -302,22 +319,29 @@ Menu_Provider()
 	esac
 
 	# Update credentials names
-	[[ $PROVIDER == 'DuckDNS' || $PROVIDER == 'FreeDNS' ]] && password='Token' || password='Password'
+	[[ $PROVIDER == 'DuckDNS' || $PROVIDER == 'FreeDNS' || $PROVIDER == 'YNDS' ]] && password='Token' || password='Password'
 }
 
 Menu_Domains()
 {
-	# Skip with FreeDNS and custom provider
+	# Skip with these providers
 	[[ $PROVIDER == 'DuckDNS' || $PROVIDER == 'No-IP' || $PROVIDER == 'Dynu' || $PROVIDER == 'OVH' ]] || return 0
 
 	G_WHIP_DEFAULT_ITEM=$DOMAINS
-	G_WHIP_INPUTBOX 'Please enter a comma-separated list of domains that shall point to this system:' || return 1
+	
```

Comment:
Trailing space, actually the empty line can probably be removed, since the `G_WHIP_DEFAULT_ITEM` belong to those input boxes anyway:
```suggestion
```